src/test/java/seedu/address/logic/wanderlustlogic/wanderlustcommands/edit/EditActivityCommandTest.java

```diff
@@ -78,8 +78,10 @@ public void execute_filteredList_success() {
 
         String expectedMessage = String.format(MESSAGE_EDIT_ACTIVITY_SUCCESS, editedActivity);
 
+
```

Comment:
Is this additional new line intentional? Sorry I'm abit OCD, can ignore if you are okay with it!

Reply:
Resolved.